packages/core/src/model.ts

```diff
@@ -84,6 +85,10 @@ export type AnimateOptions = Partial<ExtendedPosition> & {
      * New zoom level between 0 and 100
      */
     zoom?: number;
+    /**
+     * New sphere correction to apply to the panorama
```

Comment:
comment needs to be updated and the correct type is `AnimationOptions['easings']` (codestyle uses single quotes)